vendor/metisMenu/metisMenu.js

```diff
@@ -1,375 +1,354 @@
-/*
- * metismenu - v2.5.2
- * A jQuery menu plugin
- * https://github.com/onokumus/metisMenu#readme
- *
- * Made by Osman Nuri Okumuş <onokumus@gmail.com> (https://github.com/onokumus)
- * Under MIT License
- */
-
-(function (global, factory) {
-  if (typeof define === "function" && define.amd) {
-    define(['jquery'], factory);
-  } else if (typeof exports !== "undefined") {
-    factory(require('jquery'));
-  } else {
-    var mod = {
-      exports: {}
-    };
-    factory(global.jquery);
-    global.metisMenu = mod.exports;
-  }
-})(this, function (_jquery) {
-  'use strict';
-
-  var _jquery2 = _interopRequireDefault(_jquery);
+const Util = (($) => {
```

Comment:
The metis menu, that was on vendor was old. It lacked the dispose function among others. I upgraded the metisMenu with npm, but it was not being imported, so I copied the upgraded and complete version to vendor. If you check, the code is completely different, including the  dispose(), that destroys the menu, which i needed, to concatenate between models (Grouped and not grouped).